WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1318,6 +1340,14 @@ extension ReaderStreamViewController: WPTableViewHandlerDelegate {
         return UITableView.automaticDimension
     }
 
+    public func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
```

Comment:
I wish these methods weren't necessary but since the WPTableViewHandler implements them we can't just set
```
tableView.sectionHeaderHeight = 0.0 
tableView.sectionFooterHeight = 0.0 
```

Reply:
Agree! I'm not fan of having those methods overridden like that, but I couldn't find a better way.
It still surprises me that it is necessary at all.